public/window/menu.js

```diff
@@ -17,6 +18,14 @@ module.exports = (is, mainWindow, app) => {
           accelerator: 'CmdOrCtrl+,',
           click: () => mainWindow.webContents.send('toggle-settings'),
         },
+        {
+          label: 'Install Shell Command',
+          click: symlink.create,
```

Comment:
I think that these two menu items should become only one. I don't know how we could do this but, "uninstalling" is irrelevant if it's not installed yet, right?

I don't know if we could think of a way to update that menu later on, or maybe simply use the word "Toggle.." but it's not the best option I think.

Otherwise, maybe put these two items in a different menu, but as it is part of the very first menu in the menu bar, I think it matters not to overload it.

What are your thoughts on this, I'm sure it could be better, I hope it makes sense.

Reply:
I honestly didn't put much thought into it. I agree that they should be only one and it should detect wether or not there's already a symlink and show the proper option accordingly. Maybe the words could be 'Add/Remove'.

Reply:
I think it's fine to have an "Install Shell command" somewhere (not in the first menu), and check the status on startup. 

It's still okay not to update it directly I guess.

Maybe adding it to the "Tools" menu?